examples/form/src/CreateOrder.tsx

```diff
@@ -0,0 +1,72 @@
+import { useIframeHashRouter } from "hash-slash";
+import { ID } from "jazz-tools";
+import { LinkToHome } from "./LinkToHome.tsx";
+import { OrderForm } from "./OrderForm.tsx";
+import { useAccount, useCoState } from "./main.tsx";
+import {
+  BubbleTeaOrder,
+  DraftBubbleTeaOrder,
+  ListOfBubbleTeaAddOns,
+} from "./schema.ts";
+
+export function CreateOrder() {
+  const { me } = useAccount({ profile: { draft: {}, orders: [] } });
+  const router = useIframeHashRouter();
+
+  if (!me?.profile) return;
+
+  const onSave = (draft: DraftBubbleTeaOrder) => {
+    // validate if the draft is a valid order
+    const validation = draft.validate();
+    if (validation?.error) {
+      alert(validation.error);
```

Comment:
Please may you extend this to proper field errors displayed on the form. I know it means you need to figure out some state tracking for it, but I think it would be worth having a full-blown "form with validation errors" example